src/core/qgsmbtiles.h

```diff
@@ -78,11 +78,6 @@ class CORE_EXPORT QgsMbTiles
      */
     void setTileData( int z, int x, int y, const QByteArray &data );
 
-    //! Decodes gzip byte stream, returns true on success. Useful for reading vector tiles.
-    static bool decodeGzip( const QByteArray &bytesIn, QByteArray &bytesOut );
```

Comment:
We don't have any API promise to maintain here since this class wasn't part of the Python API, but I wonder whether we could keep that function as a shortcut to the ones relocated in QgsZipUtils. I'm more than happy to hear it's not needed at all :)

Reply:
Also, is there any reason this class has no python bindings? I see below you're adding the vtpk sibling, we might as well add this one?

Reply:
>  keep that function as a shortcut to the ones relocated in QgsZipUtils

I don't see a particularly good reason for that -- it's just extra bloat in a header which we don't need :)

> Also, is there any reason this class has no python bindings? I see below you're adding the vtpk sibling, we might as well add this one?

I've only exposed the vtpk ones for testing really (to avoid the need for a c++ based test, and the associated increase in compilation times). I doubt there's a compelling use case for a plugin/script to need access to these low level classes.

Reply:
@nyalldawson , all very reasonable to me.